### PR TITLE
Rework a bit the April content for step 3

### PR DESCRIPTION
**Pull request by NolwennP**
My dear Tata, here is a quick proposition of content rework for the April updates.
Tell me what you think.
The idea is to always have something meaningful at the first level of header (h1) so that in the PIM, we'll be able to use the h1 title as well as the first sentence just after and the first screenshot, as the content of the What's new panel. 😉

### CODE REVIEW

**Comment by LaureBro** [comment]
So, it's ok for me :) We'll need to deploy changes for March and Feb too right?

File: content/updates/2020-04/7-data-quality-new-improvements.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Data quality improvements
+To make sure all your products have enriched and consistent content, Data quality insights recommendations and grades are now also available for product models and for more languages.
```
Comment (NolwennP) [inline content/updates/2020-04/7-data-quality-new-improvements.md:2]:
```suggestion
To make sure all your products have enriched and consistent content, Data Quality Insights recommendations and grades are now also available for product models and for more languages.
```

File: content/updates/2020-04/1-new-screen-for-measurements-families.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# A new screen to add your own measurement units
+::: meta-data type="New" features="Measurements, Data Governance" available="mid-April" link-to-doc="../articles/manage-your-measurements.html"
+:::
+
+In order to give you more flexibility, we have improved the way we manage metrics in the PIM. Discover our new Measurements feature now!
+
+Alongside standard measurement families (eg. weight, length, volume...), create your custom measurement families per your needs (eg. inductance, luminance...) and add the units you deem necessary (Henry, Lambert...).
+
+![List measurement families](../img/Settings_Measurement_Families-updates.png)
```
Comment (NolwennP) [inline content/updates/2020-04/1-new-screen-for-measurements-families.md:9]:
```suggestion
![Measurement families list screen](../img/Settings_Measurement_Families-updates.png)
```

File: content/updates/2020-04/2-new-measurements-api-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# New dedicated API endpoints for your measurement units
```
Comment (NolwennP) [inline content/updates/2020-04/2-new-measurements-api-endpoints.md:1]:
```suggestion
# New endpoints to manage measurements
```

File: content/updates/2020-04/6-rules-engine-new-actions-and-improvements.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 # Rules engine new actions and improvements
+To extend the rules engine capabilities, we added two new actions, as well as the possibility to name your rules and improved the rules format for associations.
```
Comment (NolwennP) [inline content/updates/2020-04/6-rules-engine-new-actions-and-improvements.md:2]:
```suggestion
To extend the rules engine capabilities, we added two new actions, the possibility to name your rules and we also improved the rules format for associations.
```

File: content/updates/2020-04/7-data-quality-new-improvements.md [hunk 5]
```diff
@@ -1,4 +1,5 @@
 # Data quality improvements
+To make sure all your products have enriched and consistent content, Data Quality Insights recommendations and grades are now also available for product models and for more languages.
```
Comment (NolwennP) [inline content/updates/2020-04/7-data-quality-new-improvements.md:2]:
```suggestion
To make sure all your products have enriched and consistent content, Data Quality Insights recommendations and grades are now also available for product models. Our spell-check feature also now supports additional languages.
```

File: content/updates/2020-04/6-rules-engine-new-actions-and-improvements.md [hunk 6]
```diff
@@ -1,4 +1,5 @@
 # Rules engine new actions and improvements
+To extend the rules engine capabilities, we added two new actions, the possibility to name your rules and we also improved the rules format for associations.
```
Comment (NolwennP) [inline content/updates/2020-04/6-rules-engine-new-actions-and-improvements.md:2]:
```suggestion
Discover the latest enhancements on the rules engine: we added two new actions, the possibility to name your rules and we also improved the rules format for associations.
```

File: content/updates/2020-04/7-data-quality-new-improvements.md [hunk 7]
```diff
@@ -1,4 +1,5 @@
 # Data quality improvements
+To make sure all your products have enriched and consistent content, Data Quality Insights recommendations and grades are now also available for product models. Our spell-check feature also now supports additional languages.
```
Comment (NolwennP) [inline content/updates/2020-04/7-data-quality-new-improvements.md:2]:
```suggestion
Data Quality Insights recommendations and grades are now also available for product models to help you ensure that all your products have enriched and consistent content. Also, our spell-check feature now supports additional languages.
```

File: content/updates/2020-04/7-data-quality-new-improvements.md [hunk 8]
```diff
@@ -1,4 +1,5 @@
 # Data quality improvements
```
Comment (NolwennP) [inline content/updates/2020-04/7-data-quality-new-improvements.md:1]:
```suggestion
# Data Quality Insights made even more powerful
```

**Pull request merged**